prime-router/src/test/kotlin/fhirengine/engine/LookupTableValueSetTests.kt

```diff
@@ -77,88 +78,61 @@ class LookupTableValueSetTests {
         mockkObject(Metadata)
         every { Metadata.getInstance() } returns UnitTestUtils.simpleMetadata
 
-        var bundle = Bundle()
+        val bundle = Bundle()
```

Comment:
Updated this test to more closely resemble the conditions under which the issue was discovered and to remove mutability.

Reply:
Nice catch